tools/sanity_checks.py

```diff
@@ -321,8 +322,20 @@ def test_releases(self) -> None:
                 # Downstream ports shouldn't use transitional provides syntax
                 # FIXME: Not all wraps currently comply, only check for wraps we modify.
                 if extra_checks and patch_path:
-                    with self.subTest(step="Ports must not use 'foo = foo_dep' provide syntax; use meson.override_dependency('foo', foo_dep) and 'dependency_names = foo'.  https://mesonbuild.com/Adding-new-projects-to-wrapdb.html#overriding-dependencies-in-the-submitted-project"):
-                        self.assertEqual(self.get_transitional_provides(config), set())
+                    # Intentional leading whitespace
+                    errmsg = textwrap.dedent(f'''
+                        In the meson.build file use `meson.override_dependency({name}, {name}_dep)`
```

Comment:
We know the wrap is called `{name}.wrap` but the dependency may not be called `{name}`, and it seems confusing to imply otherwise.  The provide names are returned from `self.get_transitional_provides()`, so we could use an actual name here, but if you don't want to go to that trouble I think `foo` is better than giving a name that's potentially misleading.

Reply:
Okay, I've used `<name>` in those places.

We could do the `get_transitional_provides()` thing, but that seems like a lot of work for a fairly uncommon issue.